soepy/python/shared/shared_constants.py

```diff
@@ -1,4 +1,5 @@
 # Set values of constants used across all modules here
 MISSING_INT = -99
+INVALID_FLOAT = -99.00
```

Comment:
This does not appear to be used anywhere, can we remove it then?

Reply:
This is precisely the constant that should be used in construct_emax of solve auxiliary you commented on in issue #56. It seems I have forgotten to add it there. Adding in construct_emax, leaving it in shared_constants and resolving #56.